test/utils/runners.go

```diff
@@ -964,9 +966,39 @@ func (c *TestPodCreator) CreatePods() error {
 	return nil
 }
 
-// readRandomlyGeneratedDataFromFile reads the whole into memory at a time and then
-func readRandomlyGeneratedDataFromFile() {
```

Comment:
function name should be readTraceFromFile, the trace does not have to be randomly generated